Find a functionally equivalent code:android.widget.AbsListView.isFastScrollAlwaysVisible:COMMENT
Method Modifier: public      
Comment:/**
 * Returns true if the fast scroller is set to always show on this view.
 *
 * @return true if the fast scroller will always show
 * @see #setFastScrollAlwaysVisible(boolean)
 */

@@ -1,7 +1,7 @@
 {
-    if (mFastScroller == null) {
+    if (mFastScroll == null) {
         return mFastScrollEnabled && mFastScrollAlwaysVisible;
     } else {
-        return mFastScroller.isEnabled() && mFastScroller.isAlwaysShowEnabled();
+        return mFastScroll.isEnabled() && mFastScroll.isAlwaysShowEnabled();
     }
 }

